docs/algorithms/dagger.rst

```diff
@@ -24,21 +24,28 @@ Detailed example notebook: :doc:`../tutorials/2_train_dagger`
     :skipif: skip_doctests
 
     import tempfile
+
     import numpy as np
-    import gym
-    from stable_baselines3 import PPO
+    import seals  # noqa: F401  # needed to load "seals/" environments
     from stable_baselines3.common.evaluation import evaluate_policy
-    from stable_baselines3.common.vec_env import DummyVecEnv
-    from stable_baselines3.ppo import MlpPolicy
 
     from imitation.algorithms import bc
     from imitation.algorithms.dagger import SimpleDAggerTrainer
+    from imitation.policies.serialize import load_policy
+    from imitation.util.util import make_vec_env
 
     rng = np.random.default_rng(0)
-    env = gym.make("CartPole-v1")
-    expert = PPO(policy=MlpPolicy, env=env)
-    expert.learn(1000)
-    venv = DummyVecEnv([lambda: gym.make("CartPole-v1")])
+    env = make_vec_env(
+        "seals/CartPole-v0",
+        rng=rng,
+        n_envs=1,
```

Comment:
@AdamGleave just wanted to flag that I had to set `n_envs=1` to achieve the maximum reward. For the default settings, rewards fluctuated in the 350 to 450 range.

Reply:
nevermind. increasing the number of time_steps in dagger_trainer.train fixes that